mava/systems/tf/dial/system.py

```diff
@@ -132,6 +133,8 @@ def __init__(
                 Defaults to None.
             shared_weights (bool, optional): whether agents should share weights or not.
```

Comment:
We should perhaps just add here that when a `agent_net_config` is supplied the value of `shared_weights` is ignored, if I understand correctly?

Reply:
Thanks for the suggestions @arnupretorius :+1: I updated the code to reflect your comments.  `agent_net_config` is now changed to `agent_net_keys`.